atrium-xrpc-client/Cargo.toml

```diff
@@ -20,20 +20,27 @@ reqwest = { workspace = true, optional = true }
 surf = { workspace = true, optional = true }
 
 [features]
-default = ["reqwest-native"]
+default = ["reqwest-default-tls"]
 isahc = ["dep:isahc"]
-reqwest-native = ["reqwest/native-tls"]
-reqwest-rustls = ["reqwest/rustls-tls"]
```

Comment:
It would be useful to mention the removal of these feature flags (and addition of the new ones) in the `atrium-xrpc-client` changelog.